src/scope.rs

```diff
@@ -1811,7 +1817,6 @@ pub(crate) mod data {
 
   #[derive(Debug)]
   enum ScopeTypeSpecificData {
-    None,
```

Comment:
Does Rust use the niche with Option<ScopeTypeSpecificData>? Is it the same size as ScopeTypeSpecificData?